lib/addons/p5.dom.js

```diff
@@ -550,12 +554,19 @@
    * }
    * </code></div>
    */
+
   p5.prototype.createSelect = function(mult) {
```

Comment:
This variable name should be changed to something that could refer to either the multiple boolean value or a DOM object. Or we could leave it empty and grab the object out of arguments[].